vuln_due_date_cleanup/due_date_cleanup.py

```diff
@@ -0,0 +1,271 @@
+import requests
+import pandas as pd 
+import json
+import gzip
+import time
+import re
+import sys
+from tqdm.auto import tqdm
+from datetime import datetime, timedelta
+
+# Base URL of API Endpoint. You might have to modify this depending on your base URL.
+base_url = "https://api.kennasecurity.com/" # US AWS
+# base_url = "https://api.us.kennasecurity.com/" # US GCP
+# base_url = "https://api.ca.kennasecurity.com/" # Canada
+# base_url = "https://api.eu.kennasecurity.com/" # Europe
+
+# Checking the information provided during the script call
+if len(sys.argv) <= 3:
+        print("==========================================================================================================")
+        print("You are missing at least one parameter on your scrip call. Your script call should be like the following:")
+        print(f"{sys.argv[0]} <risk_token/API_key> <risk_meter_id> <number_of_days>")
+        print("==========================================================================================================")
+        sys.exit(1)
+elif len(sys.argv[1]) < 64:
```

Comment:
This test is great, but I would change "elif" to "if".

Reply:
Working on it.